.gitignore

```
@@ -32,7 +32,9 @@ Homestead.yaml
 npm-debug.log
 yarn-error.log
 .env
-.env.bck
+.env.*
+!.env.example
+!.env.homestead
```

```suggestion
.env*
```
Files manually added to the index aren't affected by .gitignore (and still show as changed when they're updated), so I'd just ignore all of them.